cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -32,6 +32,18 @@ spec:
   mon:
     count: 3
     allowMultiplePerNode: false
+    # A volume claim template can be specified in which case new monitors (and
```

Comment:
With #3341 there is a new example something like cluster-with-pvcs.yaml. We could add this example to that file after they come together. This cluster.yaml could remain as an example for dataDirHostPath.